data/schema.graphql

```diff
@@ -0,0 +1,22 @@
+# Person data
+type Person {
+  # The ID of an object
+  id: ID!
+  _id: Int
```

Comment:
Hmmm, why int?

Reply:
My bad, this have to be a String (cause mongo's ObjectId can be represented as a String). But, i'm consuming a public api and I don't even use mongo. Nice alert, i'll remove this field

Reply:
no problem :)
Do you know that `schema.graphql` is a generated file, right?

Reply:
anyway, the fix must come from your schema definition type: https://github.com/gsasouza/starwars-api-graphql-wrapper/blob/master/src/PeopleType.js#L22

Reply:
Yes, i used the updateSchema.js (copy-pasted from you repo) to generate schema.graphql and schema.json files. I'm following this tutorial https://facebook.github.io/relay/docs/tutorial.html

Reply:
nice job
keep going